src/test/java/com/example/bank/integration/rest/ClientRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,46 @@
+package com.example.bank.integration.rest;
+
+import com.example.bank.client.jpa.Client;
+import com.example.bank.client.jpa.ClientRepository;
+import com.example.bank.exception.ClientNotFoundException;
+import com.example.bank.integration.ClientRequestAndClientIntegrationTestUtils;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+public class ClientRepositoryIntegrationTest {
+    @Autowired
+    private ClientRepository clientRepository;
+    private final String noClientInDatabaseExceptionMessage = "Error. There is no client in database.";
+    private final Client testClient = ClientRequestAndClientIntegrationTestUtils.clientIntegrationTestBuilder();
+
+    @BeforeEach
+    void setUp() {
```

Comment:
jak juz robisz beforeEach to i powieniens zrobic afterEach zeby kasowac bo tak to bedziesz przed kazdym testem dodawal takiego samego klienta znowu do bazy.